src/wrappers/lambda.js

```diff
@@ -111,6 +114,17 @@ function baseLambdaWrapper(
                 process._events.beforeExit = originalBeforeExit;
             });
         };
+        // catch an unhandled promise rejection within the user's code
+        process.once('unhandledRejection', (reason) => {
+            utils.debugLog('Unhandled Promise Rejection caught. Please handle the rejected promise');
+            const rejectedError = {
+                name: 'UnhandledPromiseRejectionError',
+                message: reason || '',
+                stack: '',
+            };
+            eventInterface.setException(runner, rejectedError, false);
```

Comment:
Is this still needed?

Reply:
If/when we add an earlier detection, this will set to show on dashboard

Reply:
We can take it out now, but will have to be added in at that point